modules/serverless-functions-key-value-in-configmap-to-env-variable.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-functions-key-value-in-configmap-to-env-variable_{context}"]
 = Setting environment variable from a key value defined in a config map
 
-You can use the following procedure to set an environment variable from a key value defined as a config map.
+You can set an environment variable from a key value defined as a config map. A value previously stored in a config map can then be immediately used in other parts of the `func.yaml` file.
```

Comment:
Same comment here as above regarding environment variables being used at run time by the function.